convert/convert_test.go

```diff
@@ -0,0 +1,161 @@
+package convert
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/mgo.v2/bson"
+)
+
+func TestConvertBsonBytes(t *testing.T) {
+	doc := bson.M{
+		"op": "i",
+		"ns": "throttle.test",
+		"o": bson.M{
+			"_id": bson.NewObjectId(),
+			"val": "55d57fd49e8a1b0d007f73b4",
```

Comment:
Are these real ID's from something? I feel like we should have something more like `abcdef12345` which conveys alphanumeric but is obviously not a real value.